### ISSUE TITLE
Accessibility (List Block): Voice Control commands not working for editing

### ISSUE THREAD

**Issue by ezebecke** [wordpress-mobile/WordPress-iOS#19276]
### Expected behavior

To be able to control text selection/navigation/editing/deletion with Voice Control inside the List Block

### Actual behavior

Form user:

> The latest update for the iOS application broke Voice Control accessibility for list block editing.

### Steps to reproduce the behavior

Select a list item inside a list block. 
Try editing the item with Voice Control

![cUAsdA.png](https://user-images.githubusercontent.com/24919469/188231212-8c7507b7-cb77-480a-886d-fa3ac07809ce.png)

##### Tested on iPad, iOS iOS 15.6.1, WPiOS 20.5

Reported on 5527364-zd

**Comment by dcalhoun** [comment]
I was able to reproduce this issue using a 20.5 build, but I am unable to do so with the more recent releases: 24.0 and 23.9.1. Closing this issue, but please re-open it if the bug is still present.
